detox/android/detox/src/full/java/com/wix/detox/reactnative/idlingresources/NetworkIdlingResource.java

```diff
@@ -64,6 +66,12 @@ public String getName() {
         return NetworkIdlingResource.class.getName();
     }
 
+    @NotNull
+    @Override
+    public String getDescription() {
+        return "In-flight network activity";
```

Comment:
So, here, I assume we can do a bit better in the future, right? E.g., collect `runningCalls()` and compose a verbose message with pending URLs, huh?

Reply:
Exactly. I think the gap isn't that big, even.